torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3715,6 +3731,22 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            aliases=('ger', ),
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            sample_inputs_func=sample_inputs_outer,),
+    OpInfo('permute',
+           op=lambda x, dims: x.permute(dims),
```

Comment:
https://github.com/pytorch/pytorch/blob/f5a7b2e641a124751dfa3fdf1926eb4135d3aad5/aten/src/ATen/native/native_functions.yaml#L2793-L2794

Reply:
Interesting. I guess our tests assume every op has a function variant because they're calling op() directly? 

Another fix for this might be for op() to invoke the method variant when a function variant isn't available. What would you think of that approach, @kshitij12345? Are there other ops that that only have method variants?

Reply:
Separately, having a method but not a function here just seems like a UX bug. What would you think about adding the function variant in this PR, too? It would require a few other changes (like updating the docs).

Alternatively we could make that a follow-up PR, but it seems relatively straightforward to include.

Reply:
> Another fix for this might be for op() to invoke the method variant when a function variant isn't available. What would you think of that approach, @kshitij12345? 

I think it is better to manually pass the lambda (at the OpInfo entry) instead of encapsulating this detail in the OpInfo class.
This way it is very much visible that Op doesn't have a function variant.

> Are there other ops that that only have method variants?

From a very quick check, I could only find `repeat`, `expand`, `expand_as` which have this behaviour (all of them have the same note `This is method-only to match the previous tensor API. In the future we could make this a function too.`

> Alternatively we could make that a follow-up PR, but it seems relatively straightforward to include.

Have added the function variant and updated the docs.